client/state/jetpack-settings/modules/selectors.js

```diff
@@ -13,7 +13,7 @@ import { get } from 'lodash';
  * @return {?Boolean}            Whether the module is active
  */
 export function isModuleActive( state, siteId, moduleSlug ) {
-	return get( state.jetpackSettings.jetpackModules.items, [ siteId, moduleSlug, 'active' ], null );
+	return get( state.jetpackSettings.jetpackModules.items, [ siteId, moduleSlug, 'activated' ], null );
```

Comment:
Should we map the returned items at the action level to have `active` instead of `activated` to keep the consistency we were trying to maintain when we started with the redux work on the other PRs related to JP settings in Calypso ?